src/components/transcript/transcript.tsx

```diff
@@ -138,6 +141,7 @@ export class Transcript extends Component<TranscriptProps, TranscriptState> {
       // use player size to define transcript root element size
       this._setWidgetSize();
     }
+    this.props.eventManager?.listen(document, 'keydown', this._handleKeydownEvent);
```

Comment:
I don't think we have to add listener for entire document, Navigation plugin has similar feature
https://github.com/kaltura/playkit-js-navigation/blob/master/src/components/navigation/index.tsx#L293

Reply:
done